DNN Platform/Library/Services/Installer/Installers/AssemblyInstaller.cs

```diff
@@ -43,7 +45,9 @@ public class AssemblyInstaller : FileInstaller
     {
         private static readonly Regex PublicKeyTokenRegex = new Regex(@"PublicKeyToken=(\w+)", RegexOptions.CultureInvariant | RegexOptions.Compiled);
 
-		#region "Protected Properties"
+        private static readonly string OldVersion = "0.0.0.0-" + new Version(short.MaxValue, short.MaxValue, short.MaxValue, short.MaxValue);

```

Comment:
Hi @bdukes just curious, did Resharper or VS suggest to make this var static? Not that it's a problem, just wondering what's the benefit of keeping it in memory.

Reply:
It just seemed better to me that there was only one instance, instead of having to create a new `Version` every time we process an assembly.  I don't have any evidence that it makes a difference, either way.  Let me know if you'd prefer it changed back.

Reply:
No, it's fine. My personal preference would be not to create a static variable, but I don't have any evidence to support it either other than static variables are kept in memory.